DataFormats/simulation/src/Stack.cxx

```diff
@@ -365,6 +371,13 @@ void Stack::Print(Int_t iVerbose) const
   }
 }
 
+void Stack::Print(Option_t* option) const
+{
+  Int_t verbose = 0;
+  if ( option ) verbose = 1;
```

Comment:
No, The interface with Option_t* come already from TObject; the function in FairRoot  generic stack class is ok.